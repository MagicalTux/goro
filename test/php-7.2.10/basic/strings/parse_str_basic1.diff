     *** Testing parse_str() : basic functionality ***
     Basic test WITHOUT result arg
     
004- Deprecated: parse_str(): Calling parse_str() without the result argument is deprecated in %s on line %d
005- NULL
006- string(4) "val1"
007- string(4) "val2"
008- string(4) "val3"
009- 
010- Basic test WITH undefined var for result arg
011- NULL
012- array(3) {
013-   ["first"]=>
014-   string(4) "val1"
015-   ["second"]=>
016-   string(4) "val2"
017-   ["third"]=>
018-   string(4) "val3"
019- }
020- 
021- Basic test WITH existing non-array var for result arg
022- NULL
023- array(3) {
024-   ["first"]=>
025-   string(4) "val1"
026-   ["second"]=>
027-   string(4) "val2"
028-   ["third"]=>
029-   string(4) "val3"
030- }
031- 
032- Basic test with an existing array as results array
033- NULL
034- array(3) {
035-   ["first"]=>
036-   string(4) "val1"
037-   ["second"]=>
038-   string(4) "val2"
039-   ["third"]=>
040-   string(4) "val3"
041- }
042- ===DONE===
004+ Fatal error: Uncaught Error: parse_str(): Too few arguments in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/parse_str_basic1.php:12
005+ Stack trace:
006+ #0 /home/ronald/proj/goro/test/php-7.2.10/basic/strings/parse_str_basic1.php(12): parse_str(first=val1&second=val2&third=val3)
007+ #1 {main}
008+   thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/parse_str_basic1.php on line 12
