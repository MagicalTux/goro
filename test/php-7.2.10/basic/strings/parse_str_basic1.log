
---- EXPECTED OUTPUT
*** Testing parse_str() : basic functionality ***
Basic test WITHOUT result arg

Deprecated: parse_str(): Calling parse_str() without the result argument is deprecated in %s on line %d
NULL
string(4) "val1"
string(4) "val2"
string(4) "val3"

Basic test WITH undefined var for result arg
NULL
array(3) {
  ["first"]=>
  string(4) "val1"
  ["second"]=>
  string(4) "val2"
  ["third"]=>
  string(4) "val3"
}

Basic test WITH existing non-array var for result arg
NULL
array(3) {
  ["first"]=>
  string(4) "val1"
  ["second"]=>
  string(4) "val2"
  ["third"]=>
  string(4) "val3"
}

Basic test with an existing array as results array
NULL
array(3) {
  ["first"]=>
  string(4) "val1"
  ["second"]=>
  string(4) "val2"
  ["third"]=>
  string(4) "val3"
}
===DONE===
---- ACTUAL OUTPUT
*** Testing parse_str() : basic functionality ***
Basic test WITHOUT result arg

Fatal error: Uncaught Error: parse_str(): Too few arguments in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/parse_str_basic1.php:12
Stack trace:
#0 /home/ronald/proj/goro/test/php-7.2.10/basic/strings/parse_str_basic1.php(12): parse_str(first=val1&second=val2&third=val3)
#1 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/parse_str_basic1.php on line 12
---- FAILED
