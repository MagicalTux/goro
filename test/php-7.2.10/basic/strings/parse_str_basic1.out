*** Testing parse_str() : basic functionality ***
Basic test WITHOUT result arg

Fatal error: Uncaught Error: parse_str(): Too few arguments in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/parse_str_basic1.php:12
Stack trace:
#0 /home/ronald/proj/goro/test/php-7.2.10/basic/strings/parse_str_basic1.php(12): parse_str(first=val1&second=val2&third=val3)
#1 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/parse_str_basic1.php on line 12